app/assets/js/nav.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Created by student on 21.11.16.
+ */
+$(document).ready(function () {
+
+
+    offset = $('.navbar').outerHeight();
+    $('#navi li a[href^="#"]').click(function (event) {
+
+        // Prevent from default action to intitiate
+        event.preventDefault();
+
+        //remove active from all anchor and add it to the clicked anchor
+        $('#navi li a[href^="#"]').removeClass("current")
+        $(this).addClass('current');
```

Comment:
so.. in previous line you are removing class *current* from element `#navi li a[href^="#"]` and immediatelly in next line you are adding same class to same `#navi li a[href^="#"]` element? interesting.. :p